restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/BaseVoiceInterpreter.java

```diff
@@ -1588,7 +1588,8 @@ public void execute(final Object message) throws Exception {
                 }
             }
             // Start gathering.
-            final Collect collect = new Collect(restcommConfiguration.getMgAsr().getDefaultDriver(), inputType, gatherPrompts, null, timeout, finishOnKey, numberOfDigits, lang, hints);
+            final Collect collect = new Collect(restcommConfiguration.getMgAsr().getDefaultDriver(), inputType, gatherPrompts,
+                    null, timeout, finishOnKey, numberOfDigits, lang, hints, resultCallbackAttr != null && !StringUtils.isEmpty(resultCallbackAttr.value()));
```

Comment:
name of variable `resultCallbackAttr` would be more clear if it mentions partial, 
for example `partialResultCallbackAttr`